src/main/java/org/zalando/problem/spring/web/advice/AdviceTrait.java

```diff
@@ -79,7 +79,7 @@
  * @see RoutingAdviceTrait
  * @see ValidationAdviceTrait
  */
-public interface AdviceTrait {
+public interface AdviceTrait extends StatusConverter {
```

Comment:
As you want, I can rollback to `extends` way with scope `package` (or `private`) or just keep it like that! Is more about users experiences, personally I feel ok with both